### PR TITLE
Update: Changelog for 1.9.0-beta3 and prepare for release.

### PR DESCRIPTION
**Pull request by michicc**
Please check the changelog. Make yourself known if you want RC1 instead.

### CODE REVIEW

**Review by TrueBrain — DISMISSED** [review]
You have to fix a few more lines of code to get a beta3 ready:
https://github.com/OpenTTD/OpenTTD/commit/6e211908588ab5272336d0d2db3bbb4020f7004f

We should script this :)

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
+- Add: News menu entry and shortcut for deleting all messages (#7240)
+- Change: [OSX] Improved scrolling behaviour when using touchpads
+- Change: Add scrollbar to cargo legend in cargo payment rates window
+- Change: Owner of vehicle with exclusive transport rights may now load cargo from neutral stations (#7256)
+- Change: Improved UI behaviour when dragging sliders with the mouse (#7227)
+- Change: Use selected group as parent when creating a new group (#7224)
+- Change: Use SlErrorCorrupt() on pool index error when loading a savegame, instead of terminating (#7219)
```
Comment (LordAro) [inline changelog.txt:12]: this seems like a codechange, and as such shouldn't be in the changelog

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,27 @@
+1.9.0-beta3 (2019-02-24)
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
+- Add: News menu entry and shortcut for deleting all messages (#7240)
+- Change: [OSX] Improved scrolling behaviour when using touchpads
+- Change: Add scrollbar to cargo legend in cargo payment rates window
+- Change: Owner of vehicle with exclusive transport rights may now load cargo from neutral stations (#7256)
+- Change: Improved UI behaviour when dragging sliders with the mouse (#7227)
```
Comment (LordAro) [inline changelog.txt]: was there any actual changed behaviour, or was it just fixing some fallout from ...i can't remember what, some other PR?

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,27 @@
+1.9.0-beta3 (2019-02-24)
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
```
Comment (LordAro) [inline changelog.txt:4]: Seems more like a Change, imo

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+- Add: News menu entry and shortcut for deleting all messages (#7240)
+- Change: [OSX] Improved scrolling behaviour when using touchpads
+- Change: Add scrollbar to cargo legend in cargo payment rates window
+- Change: Owner of vehicle with exclusive transport rights may now load cargo from neutral stations (#7256)
+- Change: Improved UI behaviour when dragging sliders with the mouse (#7227)
+- Change: Use selected group as parent when creating a new group (#7224)
+- Change: Use SlErrorCorrupt() on pool index error when loading a savegame, instead of terminating (#7219)
+- Change: Skip reliability decay if servicing is disabled
+- Fix: Remove desert around lakes upon generation
+- Fix: Resorting file list did not update filtered rows
```
Comment (LordAro) [inline changelog.txt]: Re-sorting "resorting" is a different word...

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,29 @@
...
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
+- Add: News menu entry and shortcut for deleting all messages (#7240)
+- Change: [OSX] Improved scrolling behaviour when using touchpads
+- Change: Add scrollbar to cargo legend in cargo payment rates window
+- Change: Owner of vehicle with exclusive transport rights may now load cargo from neutral stations (#7256)
+- Change: Improved UI behaviour when dragging sliders with the mouse (#7227)
+- Change: Use selected group as parent when creating a new group (#7224)
```
Comment (LordAro) [inline changelog.txt]: s/group/vehicle group/g ?

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,29 @@
+1.9.0-beta3 (2019-02-24)
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
```
Comment (LordAro) [inline changelog.txt:5]: I'm not seeing any mention of the framerate window in beta1, should this be here?

**Review by LordAro — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 1 above)
Comment (michicc) [reply changelog.txt:12]: It's not, game crash/exit against a proper 'Corrput savegame' message.

File: changelog.txt (hunk 2 above)
Comment (michicc) [reply changelog.txt]: Dragging doesn't completely stop if the mouse leaves the slider, so behaviour change.

File: changelog.txt (hunk 4 above)
Comment (michicc) [reply changelog.txt]: I blame the commit author

File: changelog.txt (hunk 3 above)
Comment (michicc) [reply changelog.txt:4]: Marketing

File: changelog.txt (hunk 6 above)
Comment (michicc) [reply changelog.txt:5]: - Feature: Framerate display window (#6822)

File: changelog.txt (hunk 5 above)
Comment (michicc) [reply changelog.txt]: Yeah, but still, bad commit message

File: changelog.txt (hunk 2 above)
Comment (PeterN) [reply changelog.txt]: Volume sliders specifically, there are no others.

File: changelog.txt (hunk 5 above)
Comment (LordAro) [reply changelog.txt]: That hardly matters, almost all the other changelog lines are adjusted from the actual commit messages

**Review by LordAro — DISMISSED** [review]

**Review by LordAro — APPROVED** [review]

**Pull request merged**